### ISSUE TITLE
Improve `CONTRIBUTE.md` for a proper contributing guide

### ISSUE THREAD

**Issue by heysujal** [resend/react-email#483]
Currently there is no CONTRIBUTE.MD which has all the instructions on how to contribute and run the project in local and many other instructions that a open source project usually has.
The contribution instructions provided [here ](https://react.email/docs/contributing) is not enough in my opinion.

**Comment by bukinoshita** [comment]
Totally agree, let's write some contributing guides to help people contribute to this package. Thanks for opening this

**Comment by OlegGulevskyy** [comment]
Agree, because at the moment I can't even get started to dev locally, as I never dealt with turbo, it seems to fail to install packages and when it works it spawns tons of nodejs process, that uses all of free memory on Mac M1 (16Gb).
I guess I am missing some requirements to get started :(

**Comment by dotarjun** [comment]
Hey! Can I work on this?

**Comment by dotarjun** [comment]
I'd like to work on this but I think we should have a discussion on what all to take into account

**Comment by multivoltage** [comment]
@dotarjun Did you run on dev correctly?
